packages/core/integration-tests/test/schema-jsonld.js

```diff
@@ -1,26 +1,62 @@
-import {bundle, assertBundleTree} from '@parcel/test-utils';
+import { bundle, assertBundles } from '@parcel/test-utils';
+var assert = require('assert');
 
-describe.skip('schema ld+json', function() {
+describe('jsonld', function() {
   it('Should parse a LD+JSON schema and collect dependencies', async function() {
     let b = await bundle(__dirname + '/integration/schema-jsonld/index.html', {
       production: true,
       publicURL: 'https://place.holder/',
     });
+    
+    assertBundles(b, [
+      {
+        name: 'index.html',
+        assets: ['index.html']
+      },
+      {
+        type: 'css',
+        assets: ['other.css']
+      },
+      {
+        type: 'jsonld',
+        assets: ['index.html']
+      },
+      {
+        type: 'png',
+        assets: ['logo.png']
+      },
+      {
+        type: 'jpeg',
+        assets: ['image.jpeg']
+      }
+    ]);
+  });
 
-    await assertBundleTree(b, {
-      name: 'index.html',
-      assets: ['index.html'],
-      childBundles: [
-        {
-          type: 'jpeg',
-        },
-        {
-          type: 'png',
-        },
-        {
-          type: 'css',
-        },
-      ],
+  it('Should output the original json back into the index.html file inside the script tag', async function() {
+    let b = await bundle(__dirname + '/integration/schema-jsonld/index.html', {
+      production: true,
+      publicURL: 'https://place.holder/',
     });
+
+    let v1JSONLDOutputInsideScriptTag = {
+      "@context": "http://schema.org",
+      "@type": "LocalBusiness",
+      "description": "This is your business description.",
+      "name": "Parcel's parcel",
+      "telephone": "555-111-2345",
+      "openingHours": "Mo,Tu,We,Th,Fr 09:00-17:00",
+      "logo": {
+          "@type": "ImageObject",
+          "url": "images/logo.png",
+          "width": 180,
+          "height": 120
+        },
+      "image": ["images/image.jpeg", "images/image.jpeg"]
+    };
+
+    // I need to assert that the current (v2) output is the same as the v1 output
+    // aka, is there a test util that can be used to grab onto the transformed output?
+    
+    assert.ok(false);
```

Comment:
need some help here figuring out how / where I can get the output of the jsonld transformer to assert its form

Reply:
just found `outputFS, distDir` being used from the test utils, so I will be using that

Reply:
@DeMoorJasper or @mischnic is there any reason why 'chai' is not included for testing? Would I be able to add it in? No worries if not.

Reply:
No particular reason, but the builtin assert package provides everything that we need.

Reply:
🤦‍♂ you are right, i found the deep equals function I was looking for in mocha's assert -- don't know why I have always relied on chai's deep equals for that.